ydb/core/persqueue/pq_impl.cpp

```diff
@@ -936,7 +952,7 @@ void TPersQueue::CreateSupportivePartitionActor(const TPartitionId& partitionId,
     TPartitionInfo& partition = Partitions.at(partitionId);
     partition.Actor = ctx.Register(CreatePartitionActor(partitionId,
                                                         TopicConverter,
-                                                        Config,
+                                                        MakeSupportivePartitionConfig(),
```

Comment:
what happens when config is  updated? Nothing?

Reply:
the configuration of the partitions is being updated
https://github.com/ydb-platform/ydb/pull/1807/commits/0717aaeea5854e3e019fc7bf8999d7fab243abbd

Reply:
done
https://github.com/ydb-platform/ydb/pull/1807/commits/1deca5dd2c561d6ccb9aefd8a6c2eed128c02b89